src/constants/vuetorrent/Comparators.ts

```diff
@@ -33,8 +33,16 @@ const comparatorMap: Record<keyof Torrent, (a: Torrent, b: Torrent, isAsc: boole
   last_activity: (a, b, isAsc) => comparators.invertedNumeric.compare(a.last_activity, b.last_activity, isAsc),
   magnet: (a, b, isAsc) => comparators.text.compare(a.magnet, b.magnet, isAsc),
   name: (a, b, isAsc) => comparators.textWithNumbers.compare(a.name, b.name, isAsc),
-  num_leechs: (a, b, isAsc) => comparators.numeric.compare(a.num_leechs, b.num_leechs, isAsc),
-  num_seeds: (a, b, isAsc) => comparators.numeric.compare(a.num_seeds, b.num_seeds, isAsc),
+  num_leechs: (a, b, isAsc) => {
+    const primaryComparison = comparators.numeric.compare(a.num_leechs, b.num_leechs, isAsc)
+    if (primaryComparison === 0) return comparatorMap['available_peers'](a, b, isAsc)
+    else return primaryComparison
+  },
+  num_seeds: (a, b, isAsc) => {
+    const primaryComparison = comparators.numeric.compare(a.num_seeds, b.num_seeds, isAsc)
+    if (primaryComparison === 0) return comparatorMap['available_seeds'](a, b, isAsc)
+    else return primaryComparison
+  },
```

Comment:
Awesome!
I personally don't like the reference to itself, even if it requires duplication.
You can also chain fallback comparisons using the OR (`||`) operator 👀

Reply:
Fair enough. I'll opt for duplication instead as suggested. And thanks for the tip about chaining, I'll use that too when I come back to this PR to make these changes.